index.html

```diff
@@ -8,7 +8,10 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <script type="module" src="/src/js/main.js" defer></script>
+    <script type="module" src="./src/js/movies.js" defer></script>
+    <script type="module" src="./src/js/main.js" defer></script>
+    <script type="module" src="./src/js/another.js" defer></script>
```

Comment:
same here :)

Reply:
if I remove another.js which should not be there, main.js does not print the array via consol.log